src/oc/web/actions/org.cljs

```diff
@@ -205,7 +205,7 @@
 
 (defn- org-created [org-data]
   (utils/after 0
-   #(router/nav! (oc-urls/sign-up-setup-sections (:slug org-data)))))
+   #(router/nav! (oc-urls/org (:slug org-data)))))
```

Comment:
Started looking at this thinking it was ready, i see it's still WIP but noticed this. Better to use `oc-urls/all-posts` here instead of `org` since it's just `/carrot` that means the user will have to do another redirect, the other instead points to `/carrot/all-posts`.

Reply:
Rgr